README.md

```diff
@@ -61,10 +61,23 @@ You can define as many files as you like, as long as there is an output argument
 }
 ```
 
-### Options
+### PostCSS
+
+You can use PostCSS modules with watch-less-do-more incredibly easily.
+Just install the modules you want, and tell watch-less-do-more to use them in the same way you would with PostCSS.
+
+```json
+{
+  "watch-less": "watch-less-do-more -u autoprefixer -i source.less -o output.css"
```

Comment:
it'd be nice if rather than showing the entire command, you just show the options that need to be added